test/authorities/BondingCurveAuthority.t.sol

```diff
@@ -1,60 +1,200 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8;
 
-import { SetupPartyHelper } from "../utils/SetupPartyHelper.sol";
-import { Party, PartyGovernance, PartyGovernanceNFT } from "../../contracts/party/Party.sol";
-import { ProposalExecutionEngine } from "../../contracts/proposals/ProposalExecutionEngine.sol";
+import { Globals } from "../../contracts/globals/Globals.sol";
+import { Party } from "../../contracts/party/Party.sol";
+import { PartyFactory } from "../../contracts/party/PartyFactory.sol";
 import { BondingCurveAuthority } from "../../contracts/authorities/BondingCurveAuthority.sol";
-import { ArbitraryCallsProposal } from "../../contracts/proposals/ArbitraryCallsProposal.sol";
-
-contract BondingCurveAuthorityTest is SetupPartyHelper {
-    constructor() SetupPartyHelper(false) {}
+import "../TestUtils.sol";
 
+contract BondingCurveAuthorityTest is TestUtils {
```

Comment:
Why would we not use `SetupPartyHelper`?

Reply:
The Party is being deployed through `BondingCurveAuthority.createParty()`. Guess we could override `setUp()` though

Reply:
It still exposes all the other contracts needed so I think it worth it.